src/app/_components/PageSection.tsx

```diff
@@ -39,10 +39,10 @@ export function PageSection({
       <SectionDivider title={kicker} />
 
       <div
-        className={clsx(
-          'mb-6 mt-4',
-          image ? 'grid grid-cols-1 gap-6 lg:grid-cols-2' : 'max-w-readable',
-        )}
+        className={clsx('mb-6 mt-4', {
+          'grid grid-cols-1 gap-6 lg:grid-cols-2': image,
+          'max-w-readable': !image,
+        })}
```

Comment:
```suggestion
        className={clsx('mt-4', {
          'grid grid-cols-1 gap-6 lg:grid-cols-2': image,
          'mb-6 max-w-readable': !image,
        })}
```

I think we need to include `mb-6 `only when there's not an image. Otherwise we include this extra margin with the 64px gap between sections.
Example on the `security` page:
<img width="1178" alt="Screenshot 2024-09-12 at 09 38 04" src="https://github.com/user-attachments/assets/df2ddbcc-481b-4e64-a52e-70257a86cfea">